cmake/FindLua52.cmake

```diff
@@ -29,7 +29,7 @@
 find_path(LUA_INCLUDE_DIR lua.h
   HINTS
     ENV LUA_DIR
-  PATH_SUFFIXES include/lua52 include/lua5.2 include/lua-5.2 include/lua include
+  PATH_SUFFIXES include/lua52 include/lua5.2 include/lua-5.2 include include/lua51 include/lua5.1 include/lua-5.1
```

Comment:
Are there situations where Lua51 or Lua52 are installed in `/usr/include/lua` which we would break now? I think the safer bet would be to fix the version parsing and bail out on version major 5, minor != 1 or 2.